projects/ion/src/lib/message/message.component.ts

```diff
@@ -0,0 +1,35 @@
+import { Component, HostBinding, Input, OnInit } from '@angular/core';
+import { IonMessageProps } from './types';
+import { IonIconComponent } from '../icon';
+
+const icontypes = {
+  positive: 'check-solid',
+  negative_alert: 'exclamation-solid',
+  negative_error: 'close-solid',
+  warning: 'exclamation-solid',
+  info: 'info-solid',
+  custom: 'plus-solid',
+};
+
+@Component({
+  selector: 'ion-message',
+  standalone: true,
+  imports: [IonIconComponent],
+  templateUrl: './message.component.html',
+  styleUrl: './message.component.scss',
+})
+export class IonMessageComponent implements OnInit {
+  @Input() label!: IonMessageProps['label'];
+  @HostBinding('[attr.data-type]')
+  @Input()
+  type?: IonMessageProps['type'] = 'positive';
```

Comment:
```suggestion
  type: IonMessageProps['type'] = 'positive';
```
You don't need to put `?` when you already initialized the property.